typings.d.ts

```diff
@@ -0,0 +1,308 @@
+declare module 'contentful-ui-extensions-sdk' {
+
+  interface Sys {
+    id: string;
+    type: string;
+    linkType?: string;
+  }
+
+  interface SpaceMembership {
+    sys: Sys;
+    admin: boolean;
+    roles: { name: string; description: string }[];
+  }
+
+  interface ContentTypeSys {
+    sys: {
```

Comment:
Can we just define actual sys properties here and just use it in `ContentTypeAPI` instead of doing intersection?